lib/get-config.js

```diff
@@ -29,6 +34,12 @@ function resolveConfigPath(configPath, pluginPath) {
   let filePath = pluginPath || usedPath;
   // Making sure that the filePath exists
   try {
+    if (!filePath) {
+      // tests:
+      // uses an empty set of rules if no .template-lintrc is present
+      // should be able to instantiate without options
+      return {};
+    }
```

Comment:
This case is supposed to be handled in the `catch` block (that is what the `if (configPath) {} else{ return {} }` bit is for

Reply:
logic moved to catch, but with notes:

![image](https://user-images.githubusercontent.com/1360552/75094598-d4677d80-559d-11ea-9936-b1dc9315f78a.png)